Kickstarter-iOS/Views/Controllers/PledgePaymentMethodsViewController.swift

```diff
@@ -12,6 +12,11 @@ protocol PledgePaymentMethodsViewControllerDelegate: AnyObject {
     _ viewController: PledgePaymentMethodsViewController,
     didSelectCreditCard paymentSourceId: String
   )
+
+  func pledgePaymentMethodsViewControllerDidTapPledgeButton(
+    _
+    viewController: PledgePaymentMethodsViewController
```

Comment:
I know how much you love formatting comments but this is strange 😄

Reply:
-_-